Content.Client/_Starlight/Character/Info/UI/CharacterICInfoControl.xaml

```diff
@@ -8,21 +8,21 @@
         <ui1:CharacterViewControl Name="CharacterView" Access="Public"/>
         <BoxContainer Orientation="Vertical" VerticalExpand="True">
             <Label Text="Description:"/>
-            <Label Name="CharacterDesc" Access="Public" MinSize="220 100" Margin="10" VerticalAlignment="Top"
-                   HorizontalExpand="True" VerticalExpand="True" />
+            <RichTextLabel Name="CharacterDesc" Access="Public" MinSize="220 100" Margin="10" VerticalAlignment="Top"
+                   HorizontalExpand="False" VerticalExpand="True" />
```

Comment:
HorizontalExpand false is the default value, you don't need to specify it here and elsewhere, just remove the property